src/shared-components/banner/index.tsx

```diff
@@ -55,6 +59,6 @@ export const Banner = ({ content, type = 'default', onClick }: BannerProps) => {
 
 Banner.propTypes = {
   content: PropTypes.node.isRequired,
-  type: PropTypes.oneOf(['default', 'success', 'error']),
   onClick: PropTypes.func,
+  type: PropTypes.oneOf(['default', 'success', 'error']),
```

Comment:
[Nit] Looks like `danger` is also in `BannerType` but not here.

Reply:
done